presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -9107,6 +9107,10 @@ public void testPruningCountAggregationOverScalar()
     public void testAggregationWithOrderBy()
     {
         assertQuery(
+                "SELECT sum(x ORDER BY y) FROM (VALUES (1, 2), (3, 5), (4, 1)) t(x, y)",
```

Comment:
Why won't such query just fail at analysis phase? Is this because there are some automated tools that might synthetically produce such queries?

Reply:
Personally I think this is meaningless but not wrong. Similar to cases like referencing only one column after selecting all columns in a sub query. We can optimize that.  I think postgresql mentioned it as a general syntax as well:
https://www.postgresql.org/docs/current/static/sql-expressions.html#SYNTAX-AGGREGATES But I don't really have strong reasons to support one way or another.